### DISCUSSION TITLE
How to query all non-deleted comments of an item

### DISCUSSION THREAD

**Discussion by myzinsky** [directus/directus#6899] [Archive]
Right now ini could query all comments from the activity endpoint: 

```GET https://directus.instance/activity/``` with the filter ```{"action":{"_eq":"comment"}, "item":{"_eq":"6"}}```

However this shows also the deleted comments and edits of comments. Is there an api call to get the actual comments which are there right now? Or does there a kind of join exists in the filtering, such that I can remove the deleted comments?

**Comment by rijkvanzanten** [comment]
When you delete a comment, the row is actually deleted from directus_activity completely 🤔 What version of Directus are you using?

**Reply by myzinsky** [reply]
Ah okay, then no problem :D thanks
